addons/website/static/src/js/menu/content.js

```diff
@@ -3,13 +3,216 @@ odoo.define('website.contentMenu', function (require) {
+           calendarWeeks: true,
+           icons : {
+               time: 'fa fa-clock-o',
+               date: 'fa fa-calendar',
+               next: 'fa fa-chevron-right',
+               previous: 'fa fa-chevron-left',
+               up: 'fa fa-chevron-up',
+               down: 'fa fa-chevron-down',
+              },
+           locale : moment.locale(),
+           format : time.strftime_to_moment_format(l10n.date_format +' '+ l10n.time_format),
+           //widgetParent : self.$(".o_page_management_page_info_form")
+         };
+         if (self.page.date_publish) {
+            datepickersOptions.defaultDate = self.page.date_publish;
+         }
+         
+         this.$("#date_publish_container").datetimepicker(datepickersOptions);
+         
+
+         return this._super.apply(this, arguments);
+    },
+    save: function(data){        
+        var self = this;
+        var context = weContext.get();
+        var url = $(".o_page_management_page_info #page_url").val();
+        
+        var date_publish = $(".o_page_management_page_info #date_publish").val();
+        if (date_publish != "") {
+            date_publish = time.datetime_to_str(new Date(date_publish));
+        }
+        var params = {
+            id: self.page.id,
+            name: $(".o_page_management_page_info #page_name").val(), 
+            //replace duplicate following "/" by only one "/"
+            url: url.replace(/\/{2,}/g,"/"),
+            is_menu: $(".o_page_management_page_info #is_menu").prop('checked'),
+            is_homepage: $(".o_page_management_page_info #is_homepage").prop('checked'),
+            website_published: $(".o_page_management_page_info #is_published").prop('checked'),
+            create_redirect: $(".o_page_management_page_info #create_redirect").prop('checked'),
+            redirect_type: $(".o_page_management_page_info #redirect_type").val(),
+            website_indexed: $(".o_page_management_page_info #is_indexed").prop('checked'),
+            date_publish: date_publish,
+        };
+        self._rpc({
+            model: 'website.page',
+            method: 'save_page_info',
+            args: [[context.website_id], params],
+            kwargs: {
+                context: context
+            },
+        }).then(function () {
+            // If from page manager: reload url, if from page itself: go to (possibly) new url
+            if (self._getMainObject().model == 'website.page')
+                window.location.href = url;
+            else
+                window.location.reload(true);
+        });
+    },
+    _urlChanged: function () {
+        var url = this.$('input[id=page_url]').val();
+        if (url != this.page.url) {
+            this.$(".ask_for_redirect").show();
+        }
+        else {
+            this.$(".ask_for_redirect").hide();
+        }
+    },
+    _createRedirectChanged: function () {
+      var createRedirect = this.$('input[id=create_redirect]').prop('checked');
```

Comment:
work without quoting? @qsm-odoo guideline or suggest for it? 

IMHO, always use str

Reply:
You mean quoting `create_redirect`? Yes it works without but I would have quoted it too, so:
```javascript
this.$('input[id="create_redirect"]')
```
But in fact, since it is the `id` property (note: we normally use classes if possible...):
```javascript
this.$('input#create_redirect')
```